encoder_node:
  ros__parameters:
    ppr: 2048 # Pulses per revolution 
    encoder0_pins: [0, 2, 3] 
    encoder1_pins: [1, 4, 5]
    encoder2_pins: [21, 22, 23]
    encoder3_pins: [26, 27, 25]
    state_pub_fr: 24 # Publish frequency [fps]
force_estimator:
  ros__parameters:
    arm_properties:
      m: [0.03, 0.005, 0.06, 0.03, 0.01] # Link Mass [kg]
      l: [0.025, 0.025, 0.0578, 0.058, 0.045] # Link Length [m]
      k: [1.148684645825963357e-01, 1.733173644303289129e-01, 6.413511406290881012e-02, 3.144847601149513422e-02] # Joint stiffness [Nm / rad]
      r: [0.02, 0.02, 0.015] # Distance of tendon to arm center [m]
      f: 0.75569389 # Constant Tendon force [Nm]
